0.wave1_data/3.process_features/extract_sc_interstellar.ipynb

```diff
@@ -0,0 +1,771 @@
+       "      <th>treatment</th>\n",
+       "      <th>dose</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>A1</td>\n",
+       "      <td>A1</td>\n",
+       "      <td>1</td>\n",
+       "      <td>1</td>\n",
+       "      <td>1</td>\n",
+       "      <td>LPS</td>\n",
+       "      <td>10µg/ml</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>B1</td>\n",
+       "      <td>B1</td>\n",
+       "      <td>2</td>\n",
+       "      <td>1</td>\n",
+       "      <td>2</td>\n",
+       "      <td>LPS</td>\n",
+       "      <td>1µg/ml</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>C1</td>\n",
+       "      <td>C1</td>\n",
+       "      <td>3</td>\n",
+       "      <td>1</td>\n",
+       "      <td>3</td>\n",
+       "      <td>H2O2</td>\n",
+       "      <td>500µM</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>D1</td>\n",
+       "      <td>D1</td>\n",
+       "      <td>4</td>\n",
+       "      <td>1</td>\n",
+       "      <td>4</td>\n",
+       "      <td>H2O2</td>\n",
+       "      <td>50µM</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>E1</td>\n",
+       "      <td>E1</td>\n",
+       "      <td>5</td>\n",
+       "      <td>1</td>\n",
+       "      <td>5</td>\n",
+       "      <td>ATP</td>\n",
+       "      <td>1mM</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  well wellName  row  col  alias treatment     dose\n",
+       "0   A1       A1    1    1      1       LPS  10µg/ml\n",
+       "1   B1       B1    2    1      2       LPS   1µg/ml\n",
+       "2   C1       C1    3    1      3      H2O2    500µM\n",
+       "3   D1       D1    4    1      4      H2O2     50µM\n",
+       "4   E1       E1    5    1      5       ATP      1mM"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Load platemap file\n",
+    "platemap_df = pd.read_csv(platemap_file)\n",
+    "platemap_df.head()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "72daff37",
+   "metadata": {},
+   "source": [
+    "## Set up `SingleCells` class from Pycytominer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "06005196-3d7b-4c71-a1c7-c767ee38fb8e",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/jenna/anaconda3/envs/3.process_features_interstellar/lib/python3.8/site-packages/pycytominer/cyto_utils/util.py:61: UserWarning: Non-canonical compartment detected: per_dilatednuclei, per_translocatednuclei, per_nuclei\n",
```

Comment:
Depending on how important it might be to see this within the context of the notebook or output from this repo, consider suppressing the warning here and leaving a comment as to why the warning was suppressed. Doing this might make the code more explainable and increase readability of the output.

Reply:
I am honestly not sure what this warning is at all. I am going to leave it since suppressing it without any info wouldn't be good. I do agree that if I knew, this is a great idea.